woodwork/tests/accessor/test_serialization.py

```diff
@@ -601,6 +647,54 @@ def test_to_disk_with_latlong(latlong_df, tmpdir, file_format):
         assert latlong_df.ww.schema == deserialized_df.ww.schema
 
 
+def test_to_disk_parquet_no_file_extension(sample_df, tmpdir):
+    if _is_dask_dataframe(sample_df) or _is_spark_dataframe(sample_df):
+        pytest.skip(
+            "Specifying filename for writing Dask or Spark DataFrames to parquet is not supported."
+        )
+    sample_df.ww.init(index="id")
+    sample_df.ww.to_disk(str(tmpdir), filename="parquet_data", format="parquet")
+
+    error_msg = "Could not determine format. Please specify filename and/or format."
+    # Without specifying format, WW doens't know what type of file this is
+    with pytest.raises(ValueError, match=error_msg):
+        deserialized_df = read_woodwork_table(
+            str(tmpdir),
+            filename="parquet_data",
+        )
+
+    deserialized_df = read_woodwork_table(
+        str(tmpdir),
+        filename="parquet_data",
+        format="parquet",
+    )
+    pd.testing.assert_frame_equal(
+        to_pandas(sample_df, index=sample_df.ww.index, sort_index=True),
+        to_pandas(deserialized_df, index=deserialized_df.ww.index, sort_index=True),
+    )
+
+
+def test_to_disk_parquet_typing_info_file_is_none(sample_df, tmpdir):
+    sample_df.ww.init(index="id")
+    sample_df.ww.to_disk(str(tmpdir), format="parquet")
+
+    filename = "data.parquet"
+    format = None
+    if _is_dask_dataframe(sample_df) or _is_spark_dataframe(sample_df):
+        filename = None
+        format = "parquet"
+    deserialized_df = read_woodwork_table(
+        str(tmpdir),
+        filename=filename,
+        typing_info_filename=None,
+        format=format,
+    )
+    pd.testing.assert_frame_equal(
+        to_pandas(sample_df, index=sample_df.ww.index, sort_index=True),
+        to_pandas(deserialized_df, index=deserialized_df.ww.index, sort_index=True),
+    )
+
+
```

Comment:
I think it'd be beneficial to do a test where we explicitly check that the `ww_meta` in the parquet file metadata matches what we expect given some dataframe that we've initialized and altered. This would give us a good indication on whether or not the serializer code works as we'd expect.

Reply:
Added a new test in  for this purpose.